sorbet/rbi/shims/opentelemetry-sdk.rbi

```diff
@@ -62,6 +62,12 @@ module OpenTelemetry
     class TracerProvider
       sig { params(name: T.nilable(String), version: T.nilable(String)).returns(Tracer) }
       def tracer(name = nil, version = nil); end
+
+      sig { params(timeout: T.nilable(Numeric)).void }
+      def shutdown(timeout: nil); end
+
+      sig { params(timeout: T.nilable(Numeric)).void }
+      def force_flush(timeout: nil); end
```

Comment:
Note to self: I need to upstream these too https://github.com/Shopify/rbi-central/pull/189